cmd/vault-manager/main.go

```diff
@@ -111,14 +132,18 @@ func main() {
 				err = toplevel.Apply(config.Name, address, dataBytes, dryRun, threadPoolSize)
 				if err != nil {
 					fmt.Println(fmt.Sprintf("SKIPPING REMAINING RECONCILIATION FOR %s", address))
+					instanceSuccesses[address] = 1
```

Comment:
nit: you could report the metric directly instead of collecting it in a map

Reply:
There was a reasoning for this approach but unsure if its valid. Is it preferable (does it matter) for all metrics/label pairings to be collected within the same scrape?  I opted for an implementation that would expose all the metrics at roughly the same time.

Reply:
wouldn't that imply that potential fails would be exposed rather late depending on how long all reconciliations would take? by exposing them immediately they are available for prometheus to scrape and alert.

to my knowledge there is no need for metrics to show up at the same time at the scrape endpoint

Reply:
Great point. I will adjust